### DISCUSSION TITLE
[UI] Allow for clicking outside of a modal to dismiss it

### DISCUSSION THREAD

**Discussion by r-c-k** [iotaledger/firefly#1553] [Features / Enhancements]
## Description

Additionally to the "Cancel" and "X" buttons, clicking outside of a modal should dismiss it.

## Motivation

Convenience and alignment with user expectations.

## Are you planning to do it yourself in a pull request?

No.

<img width="1392" alt="Bildschirmfoto 2021-04-11 um 21 06 55" src="https://user-images.githubusercontent.com/17240275/114318027-4a64d600-9b0b-11eb-8ab6-bc4d7175f120.png">

**Comment by nicole-obrien** [comment]
Thank you for the suggestion this will be added in the next release 🔥
